README.md

```diff
@@ -86,3 +86,32 @@ Currently the `_all`, `_default`, `_extra` and `_custom` special field shortcuts
 supported.
 
 Custom fields are not currently returned.
+
+## Retrieving comments by ID
+
+```javascript
+// .getComment() accepts one parameter, ID of comment, as number
+let comment = await api.getComment(123456);
+```
+
+Returned object has one property, id of retrieving comment, which contains object of comments data.
+
+## Retrieving all comments of bug
+
+```javascript
+// .getComments() accepts one parameter, ID of bug, as number
+let comment = await api.getComments(123456);
```

Comment:
I think this API would be nicer to callers if it just returned the Comment object instead of the object containing it.

Reply:
That's a good point, done.